app/uk/gov/hmrc/apiplatformtest/views/definition.scala.txt

```diff
@@ -45,10 +45,15 @@
               "hint": "This is your Call back URL.",
               "shortDescription": "Callback URL",
               "validation": {
-                "errorMessage": "Callback URL must be a valid URL",
+                "errorMessage": "Callback URL must be a valid https URL",
                  "rules": [
                    {
                      "UrlValidationRule": {}
+                   },
+                   {
+                     "RegexValidationRule": {
+                       "regex" : "^https://.*$"
```

Comment:
Some chars may need to be escaped. Try "^https\:\/\/.*$"

Reply:
I didn't think colon and forward slash needed escaping?

Reply:
Decided to make it simpler a remove any need for escaping! :-)